aws-s3-public-bucket/variables.tf

```diff
@@ -0,0 +1,46 @@
+variable "owner" {
+  type        = string
+  description = "Owner for tagging and naming."
+}
+
+variable "project" {
+  type        = string
+  description = "Project for tagging and naming."
+}
+
+variable "service" {
+  type        = string
+  description = "Service for tagging and naming."
+}
+
+variable "env" {
+  type        = string
+  description = "Env for tagging and naming."
+}
+
+variable "bucket_name" {
+  type        = string
+  description = "The name of the bucket."
+}
+
+variable "allow_public_list" {
+  type        = bool
+  description = "Allow public to list bucket contents."
+  default     = false
+}
+
+variable "bucket_policy" {
+  type        = string
+  description = "A policy to attach to this bucket."
+  default     = ""
+}
+
+variable "bucket_contents" {
+  type        = string
+  description = "Sentence that describes the data inside this bucket"
+}
+
+variable "public_reason" {
```

Comment:
`public_read_justification`? maybe? I feel like there is a more precise name we can use here, what do you think?

Reply:
I think this makes sense! Will rename